backend/dataall/modules/datasets/aws/kms_dataset_client.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from dataall.base.aws.sts import SessionHelper
+from botocore.exceptions import ClientError
+
+
+log = logging.getLogger(__name__)
+
+
+class KmsClient:
+    _DEFAULT_POLICY_NAME = 'default'
+
+    def __init__(self, account_id: str, region: str):
+        session = SessionHelper.remote_session(accountid=account_id, region=region)
+        self._client = session.client('kms', region_name=region)
```

Comment:
since it's a new part, may be we can start using 'dependency injection patter' as we discussed before?
Smth like
```
  def __init__(self, account_id: str, region: str, client = None):
        self._account_id = account_id
        self.region = region
        if client is not None:
             self._client = client
        else:
             session = SessionHelper.remote_session(accountid=account_id, region=region)
             self._client = session.client('kms', region_name=region)
```

Reply:
I will do it on a separate PR, to study how to extend it to other boto3 clients. I think we can look for a way of injecting without doing the distinction between tests and no-tests. 

Before we call the KmsClient we would create the session and then pass it directly, that way we do not need the if-else. 
```
session = SessionHelper.remote_session(accountid=account_id, region=region)
boto_client = session.client('kms', region_name=region)
kms_client = KmsClient(client=boto_client)
```

```
  def __init__(self, account_id: str, region: str, client: boto3.session.Session):
        self._account_id = account_id
        self.region = region
        self._client = client
```

But creating the boto_client every time we initialise the KmsClient seems a bit painful, so I need to look for a better way of doing that. That's why I want a separate PR :)